src/callback/file_callbacks.cpp

```diff
@@ -65,7 +65,7 @@ namespace Callbacks{
                 file->seek(current_request->offset);
 
                 //We read and store the requested data in an array.
-                godot::PoolByteArray buffer {file->get_buffer(current_request->sizebytes)};
+                godot::PackedByteArray buffer {file->get_buffer(current_request->sizebytes)};
                 int size {buffer.size()};
                 const uint8_t* data {buffer.read().ptr()};
```

Comment:
@piiertho @2shady4u no clue how to fix this, as `read()` is not available on the `PackedByteArray `type (which replaced Godot 3's `PoolByteArray`). Maybe you have some ideas.

Reply:
I don't see any sort of "read"-equivalent proxy type like there is for the `VectorWriteProxy`. I think you should be able to just call `.ptr()` to directly fetch the `const uint8_t*` for the buffer.

Reply:
Fixed! Thanks a lot.